src/System.Diagnostics.Process/tests/ProcessTests.cs

```diff
@@ -179,7 +181,7 @@ public void ProcessStart_UseShellExecute_OnWindows_DoesNotThrow(bool isFolder)
                     }
                     else
                     {
-                        Console.WriteLine("Warning: Need to investigate why process is null when opening file on this machine. Refer to #24048");
+                        // sometimes process is null: tracked by #24048
```

Comment:
Delete the whole else block and move the comment `// sometimes process is null: tracked by #24048` to the `if (px != null)` line. Maybe put it inline or above.